APR-Core/zones/Horde.lua

```diff
@@ -195,44 +163,26 @@ APR.QuestStepListListing["SpeedRun"] = {
 	["1543-Z13-TheMaw-Story"] = "Z-13-The Maw-Story",
 	["1525-Z14-Revendreth-Story"] = "Z-14-Revendreth-Story",
 	["1671-Z15-Oribos-Story"] = "Z-15-Oribos-Story",
-	["1533-Bastion-NonStoryMode-1"] = "Test1",

```

Comment:
Why did you only keep shadowland on the Speedrun route ?

Reply:
not 100% sure if it still needed for the speed run or not safer to leave that till someone can rest without it